parsl/dataflow/task_status_poller.py

```diff
@@ -35,6 +35,20 @@ def status(self) -> Dict[object, JobStatus]:
     def executor(self) -> ParslExecutor:
         return self._executor
 
+    def scale_in(self, n):
+        ids = self._executor.scale_in(n)
+        if ids is not None:
+            for id in ids:
+                del self._status[id]
+        return ids
+
+    def scale_out(self, n):
+        ids = self._executor.scale_out(n)
```

Comment:
Is None returned if there was a scale_out error ? Just trying to understand if anything needs to happen here if the scale out failed.